src/common/ExportSettingsModal/ExportSettingsModalContainer.js

```diff
@@ -39,16 +43,26 @@ const ExportSettingsModalContainer = ({
 
       setIsExporting(false);
 
-      return exportCsv(exportData, {});
+      onCancel();
+
+      return exportCsv(
+        [{ ...EXPORT_ORDER_FIELDS, ...EXPORT_LINE_FIELDS }, ...exportData],
```

Comment:
does it mean that all columns included always in the header? IMHO, it's controlled by user and fields he selects, no?

Reply:
IIUC, it's because of `onlyFields`